Common/Orders/OrderTypes.cs

```diff
@@ -68,6 +68,12 @@ public enum OrderDuration
         /// </summary>
         Day
         */
+
+        /// <summary>
+        /// Order valid until specific type.
+        /// </summary>
+        Specific
```

Comment:
this would need to be implemented in the engine. I'm thinking long term this enumeration is an API convenience thing, but the raw data would be a date to expire? thoughts?